app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class LikesController < ApplicationController
+  def index; end
+
+  def show; end
+
+  def new; end
+
+  def edit; end
```

Comment:
**OPTIONAL** - You have done well so far, but you have actions in your controller that aren't configured in your routes, I suggest you remove them or configure them in your routes. Please fix this 🙏